index.md

```diff
@@ -187,4 +220,47 @@ Some journals, for example, *Annual Reviews* lets the authors retain the right t
 If an article is an adaptation of a previously published article, it must be noted in the cover letter, and major changes noted. 
 Evaluating whether such changes constitute a significant revision will be part of the review process.
 
+## Author Order
+Authors should, to the extent possible, determine the author order
+among themselves.  Each work must have a section describing the actual
+contributions of authors (and of those acknowledged) to provide
+clarity, and journal templates include such a section.  
+<!-- MRS: still need to add that section of the templates.  We should also likely provide more explicit instructions here as to how finely to divide the contributions. -->
```

Comment:
How about just, "Please note, since this is an electronic-only journal, there is no length limit when you describe the authors' contributions, so we recommend describing what they actually did rather than simply categorizing their roles as might be done in more conventional journals."? You could also add, if you like, "This provides a mechanism to clearly indicate how and what each author contributed for posterity."

Reply:
(I wouldn't necessarily want to give rules on this, though I think it might be appropriate to link to some good examples when we have them... I just don't see how to make rules on it at present.)

Reply:
Adjusted (mostly what you asked for).